commerce_2checkout/commerce_2checkout.module

```diff
@@ -172,6 +172,10 @@ function commerce_2checkout_redirect_form_validate($order, $payment_method) {
     commerce_order_status_update($order, 'completed');
     commerce_order_save ($order);
     // Save the transaction information.
+
+    // Allow modules to alter parameters of the API response.
+    drupal_alter('commerce_2checkout_validate_form_data', $_REQUEST, $order);
```

Comment:
Can you provide an example scenario where altering the passback would be useful?